client/src/components/ProfileDetail/ProfileDetail.tsx

```diff
@@ -0,0 +1,57 @@
+import { Avatar, Card, CircularProgress, Grid, Paper, Typography } from '@mui/material';
+import { useAuth } from '../../context/useAuthContext';
+import React from 'react';
+import { useStyles } from './useStyles';
+import { LocationOn } from '@mui/icons-material';
+
+const ProfileDetail = (): JSX.Element => {
+  const classes = useStyles();
+  const { profile } = useAuth();
+
+  const title = () => {
+    return profile.accountType === 'pet_owner' ? 'Kind pet owner' : 'Loving pet sitter';
+  };
+
+  return profile ? (
+    <Paper elevation={8} className={classes.paper}>
```

Comment:
You can use the Card API from MUI since we are displaying information about a particular topic

Ref: https://mui.com/api/card/